spec/mailers/previews/unsubmitted_course_alert_mailer_preview.rb

```diff
@@ -17,4 +18,12 @@ def alert
   def example_user
     User.new(email: 'sage@example.com', username: 'Ragesoss')
   end
+
+  def create_classroom_program_manger
+    unless SpecialUsers.classroom_program_manager
+      SpecialUsers.define_singleton_method(:classroom_program_manager) do
```

Comment:
This could possibly cause problems in production, I think, because the singleton method would stay in place and affect subsequent requests. It wouldn't hit this code in Wiki Education production, but it would on Programs & Events Dashboard where no classroom_program_manager is assigned.

Maybe instead, modify the mailer so that it can still work when classroom_program_manager is `nil`.

Reply:
If the Rails mailer can handle `reply_to: nil`, then you could just use safe navigation. That user isn't actually the sender, just the reply-to email, so leaving out the reply-to would fall back to the default of replying to the sender. (In production, the sender is dashboard@dashboard.wikiedu.org .)